### PR TITLE
Theme Submission: hugo-eureka

### PR DESCRIPTION
**Pull request by wangchucheng**

### CODE REVIEW

File: content/theme/hugo-eureka.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+demo: https://themes.gohugo.io/theme/hugo-eureka/
+author: C. Wang
+date: 2020-11-22 # Enter the date you submitted the theme YYYY-MM-DD
+github_branch: master # Specify the repos default branch
+ssg:
+  - Hugo
+cms:
+  - No CMS
+css:
+  - Tailwindcss
```
Comment (JugglerX) [inline content/theme/hugo-eureka.md:13]: This should be `Tailwind`

File: content/theme/hugo-eureka.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
+---
+title: "Eureka"
+github: https://github.com/wangchucheng/hugo-eureka
+demo: https://themes.gohugo.io/theme/hugo-eureka/
+author: C. Wang
+date: 2020-11-22 # Enter the date you submitted the theme YYYY-MM-DD
```
Comment (JugglerX) [inline content/theme/hugo-eureka.md:6]: Please remove comments from front-matter

File: content/theme/hugo-eureka.md [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+github_branch: master # Specify the repos default branch
+ssg:
+  - Hugo
+cms:
+  - No CMS
+css:
+  - Tailwindcss
+archetype:
+  - Posts
```
Comment (JugglerX) [inline content/theme/hugo-eureka.md:15]: This isn't the specific Hugo archetypes, its more the type of website. You can use `Documentation` and `Blog` here

**Review by JugglerX — CHANGES_REQUESTED** [review]
@wangchucheng thank you for you submission, just a few minor things to update and I can merge it, thanks!

**Comment by wangchucheng** [comment]
@JugglerX Hi, I have made some changes to fix those. If there are still some problems, please don't hesitate to tell me. Thank you for your time!

**Review by JugglerX — APPROVED** [review]

**Pull request merged**